index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Google</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
+</head>
+<body class="font-roboto">
+    <nav class="w-full gap-6 px-10 py-3 flex justify-end items-center">
+        <div class="cursor-pointer text-sm font-light hover:underline">Gmail</div>
+        <div class="cursor-pointer text-sm font-light hover:underline">Images</div>
+        <div class="cursor-pointer w-9 h-9 rounded-full hover:bg-gray-200 flex justify-center items-center">
+            <i class="bi bi-grid-3x3-gap-fill"></i>
+        </div>
+        <button class="bg-blue-600 text-white outline-none px-4 py-2 rounded-md hover:shadow-md font-medium">Sign in</button>
+    </nav>
+    <main class="flex flex-col items-center min-h-[80vh]">
+        <img src="./img/logo.png" alt="" class="w-52 lg:w-68">
+        <div class="border border-gray-300 rounded-3xl relative flex items-center px-4 mt-10 w-[90%] lg:w-2/5 hover:shadow-md">
+            <i class="bi bi-search text-gray-400 text-sm"></i>
+            <input type="text" class="outline-none bg-none border-none py-[10px] ml-3 w-full">
```

Comment:
no abstract classes

Reply:
py-[10px] is an abstract class and it is not allowed you should do this in config file